extensions/cal-com-share-meeting-links/src/index.tsx

```diff
@@ -25,6 +25,26 @@ export default function Command() {
 
   return (
     <List isLoading={isLoading}>
+      <List.Item
+        title="Open Cal.com Dashboard"
+        actions={
+          <ActionPanel>
+            <Action.OpenInBrowser url="https://app.cal.com" />
+          </ActionPanel>
+        }
+      />
+      {user && (
+        <List.Item
+          title="Copy My Link"
+          subtitle={"/" + user.username}
+          actions={
+            <ActionPanel>
+              <Action.CopyToClipboard title="Copy My Link" content={`https://cal.com/${user.username}`} />
+              <Action.OpenInBrowser url={`https://cal.com/${user.username}`} />
+            </ActionPanel>
+          }
+        />
+      )}
```

Comment:
I don't think this is a good UI/UX - maybe it's better to add as action panel items?

Reply:
No worries, I was just copying the setup of the [Calendly extension](https://github.com/raycast/extensions/blob/176da216c93bbf1cc9031d987173ded7e63f57b0/extensions/calendly/src/calendly.tsx) to be honest. Happy to create actions for these instead, although arguably it's harder to get to those links through actions because you have to remember the shortcuts.

Reply:
You could also create teo sections, one with the items and one with above - I don't think they should be in top at least